File: Backend/src/main/java/com/example/backend/service/AvailabilityService.java
package com.example.backend.service;

import com.example.backend.dto.availability.AvailabilityRequestDTO;
import com.example.backend.dto.availability.AvailabilityResponseDTO;

import java.util.List;

public interface AvailabilityService {
    List<AvailabilityResponseDTO> findAllAvailability(Long businessId);

    AvailabilityResponseDTO updateAvailability(AvailabilityRequestDTO availabilityRequestDTO, Long userId);
}

File: frontend/src/services/AvailabilityService.js
import api from "./api.js";

const AvailabilityService = {
    findAllAvailabilityByBusinessId: async ({businessId}) => {
        return await api.get(`/api/availability/${businessId}`)
    },
    updateAvailability: async (data) => {
        return await api.put(`/api/availability/edit`,data)
    }

}
export default AvailabilityService;